.github/workflows/labeler.yml

```diff
@@ -0,0 +1,14 @@
+name: Pull Request Labeler
+on:
+- pull_request_target
+
+jobs:
+  triage:
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@v4
+      with:
+        sync-labels: true
```

Comment:
The `sync-labels` option means that if the pull request is updated, then the labels will be updated too.